src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js

```diff
@@ -0,0 +1,203 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var templates = {
+            copyLocales: function(item) {
+                var template = [
+                    '<div class="copy-locales-overlay-content">',
+                    '   <label>',
+                    this.sandbox.translate('content.contents.settings.copy-locales.copy-from'),
```

Comment:
@wachterjohannes Wouldn't it be better to make the templates as strings not functions and pass the needed text via the underscore-templating method?